src/main/java/com/twilio/oai/TwilioNodeGenerator.java

```diff
@@ -250,6 +252,18 @@ private void addDependent(final Map<String, Object> dependents, final String dep
         dependent.put("filename", dependentName);
     }
 
+    private Optional<CodegenModel> resolveComplexType(Optional<CodegenModel> item) {
+        for (CodegenProperty prop: item.get().vars) {
+            if (prop.dataType.equals("Array<TestResponseObjectTestArrayOfObjects>")) {
+                prop.dataType = "Array<object>";
+            }
+            else if (prop.dataType.equals("TestResponseObjectTestObject")) {
```

Comment:
This is too specific to the test fixture (e.g., if we change the name of the component in the fixture then this no longer works. Can we inspect the properties of the property to determine if it's a complex type?